src/Assets/analytics.ts

```diff
@@ -0,0 +1,7 @@
+export default function dataLayerPush(obj: any) {
```

Comment:
What do you think about calling this `addToDataLayer`?

Reply:
It is called a dataLayerPush in GTM.